.github/actions/gcp-auth/action.yaml

```diff
@@ -0,0 +1,66 @@
+name: Authenticate against GAR
```

Comment:
there is an inconsistency, `gcp-auth` vs `Authenticate against GAR`
is this action only working for GAR, or for GCP in general?